dot/rpc/modules/author.go

```diff
@@ -76,8 +76,8 @@ type decodedKey struct {
 type ExtrinsicStatus struct {
 	IsFuture    bool
 	IsReady     bool
-	Isfinalised bool
-	Asfinalised common.Hash
+	IsFinalized bool
+	AsFinalized common.Hash
```

Comment:
Are we certain about the non UK spelling?

Reply:
yeah, I was seeing an error in the front end console due to not being able to find "finalised" and expecting "finalized', substrate uses `finalized` so for things that require interop we need to use `finalized`

Reply:
:smile: yes that's true. I think `finalized` is lint-ignored anyway!